src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializer.cs

```diff
@@ -404,6 +404,15 @@ public static int WriteTags(byte[] buffer, int cursor, Metric metric, ReadOnlyTa
             buffer[cursor++] = unchecked((byte)',');
         }
 
+        if (metric.MeterTags != null)
```

Comment:
I see that the current tests have been updated to validate `MeterTags`. Is there a test that validates null `MeterTags`?"